### PR TITLE
Pause menu&portal cutscene

### PR DESCRIPTION
**Pull request by tulubin**
Added a simple pause menu
*There is an existed error in playerRig prefab that blocks all updates to it.

Adjusted main camera to be more like an action game

Added Timeline for the portal cutscene

### CODE REVIEW

File: Assets/GlitchEscape/Scripts/Interaction/InteractablePortal.cs [hunk 1]
```diff
@@ -55,6 +60,7 @@ public void OnPlayerExitInteractionRadius(Player player)
 
     public void OpenPortal()
     {
-        this.transform.parent.gameObject.SetActive(true);
+        portalCutscene.Resume();
+        // this.transform.parent.gameObject.SetActive(true);
     }
 }
```
Comment (rbaua) [inline Assets/GlitchEscape/Scripts/Interaction/InteractablePortal.cs]: Player can keep moving during cutscenes -  freeze player while cutscene is happening.

File: Assets/GlitchEscape/Config/Input/Input.cs [hunk 2]
```diff
@@ -905,6 +913,39 @@ public @Input()
...
+                    ""processors"": """",
+                    ""groups"": ""Gamepad"",
+                    ""action"": ""PauseMenu"",
+                    ""isComposite"": false,
+                    ""isPartOfComposite"": false
+                },
+                {
+                    ""name"": """",
+                    ""id"": ""aa6ffb8b-438a-46eb-8635-fc92f59dc1c2"",
+                    ""path"": ""<Keyboard>/h"",
```
[the remark this answers is not in the document]
Comment (tulubin) [reply Assets/GlitchEscape/Config/Input/Input.cs]: Right, that was for testing, forget to remove

File: Assets/GlitchEscape/Scripts/Interaction/InteractablePortal.cs (hunk 1 above)
Comment (tulubin) [reply Assets/GlitchEscape/Scripts/Interaction/InteractablePortal.cs]: This is a know issue, however I don't really want touch other scripts in this testing stage. Could be easily solved later.

**Comment by SeijiEmery** [comment]
nevermind, reverted this PR: https://github.com/glitch-escape/glitch-escape/pull/124

**Comment by SeijiEmery** [comment]
outstanding issues:

- event system needs to be part of the player rig prefab!!!! (why do you think it's acceptable to push a new feature (ie. menus) and NOT have it work across all scenes????)
- missing script on player prefab (wtf, when did that happen?), though that doesn't seem to be your fault
- menu includes a settings button (that doesn't do anything), but does NOT include a quit game or restart level button. THIS WAS THE ENTIRE POINT OF ADDING THIS MENU IN THE FIRST PLACE. Trivial fix - fix this.
- menu looks terrible, but this is just programmer art / not necessarily your fault
- any null reference errors (ie. script breakage) that result across ANY other scenes that result from a feature that you added are your responsibility to fix. That means you need to get missing dependencies (ie. null references) automatically (ie. `GameObject.FindObjectOfType<T>`) and/or shut off code branches when that occurs. I'll put out a style guide or something to clarify this.

**Pull request merged**